feature-libs/storefinder/components/store-finder-store-description/store-finder-store-description.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="location">
   <div class="container">
     <div class="row">
-      <article class="cx-store col-md-4">
+      <article class="cx-store col-lg-4">
```

Comment:
Class changes/removals are also considered breaking changes so we still need to address this here.

NOTE: The version these changes will go into will be different than `6.8`, but unfortunately we still don't know what it will be. Just a heads up if you weren't already aware.

Reply:
Done. I have left the version as 6.8 for now, once we have the proper number it will probably need to be changed in all the tickets with feature flags.